_posts/2017/2017-03-22-microxchg-2017-microservices-conference.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "microXchg 2017 - Microservices conference in Berlin"
+date: "2017-02-22 12:02:22"
+image: blog-header/microxchg-2017.jpg
+categories: events
+authors: ["Benjamin"]
+---
+
+# MicroXchg 2017 - Microservices conference in Berlin
+
+At mid of February the [microXchg conference](http://microxchg.io) took place in Berlin.
+It is the biggest two day event in Europe that focuses on Microservices - an architectural design of software systems that has generated a lot of traction within the last years as it is capable of transforming SOA and monolith systems.
+
+## Kickoff Conference Panel - on evening before
+
+One day before the [Microservices Meetup Berlin](https://www.meetup.com/Microservices-Meetup-Berlin/) held a [conference panel](https://www.meetup.com/Microservices-Meetup-Berlin/events/237309001/) event with a plenum discussions.
+Invited were some of the greatest influencers in Microservices these days: Uwe Friedrichsen, Oliver Gierke, Eberhard Wolff and Fred George.
+Although I learned not really something new totally new, there where quite a lot of more details.
+Especially Fred's tellings of histories of his life and big, big software projects were worth every second I could spend sitting in the first row.
+I also had the opportunity to ask questions and chat with participants.
+
+## Workshop Day
+
+I visited the workshop building microservcies with Scala and Akka by [Yaroslav Tkachenko](http://sap1ens.com).
+It was an awesome experience.
+He is a highly skilled professional with a very long background in Scala and prepared a workshop that could have easily spanned two more days.
+
+The material until the very last seconds of the workshop visited the conference as well [afterwards](http://sap1ens.com/blog/2017/02/26/microxchg-2017), where we had many interesting discussions about the tech communities in Canada.
+
+## Visited Talks
+
+I could visit a lot of interesting sessions with a wide variety of topics.
+All talks where recorded with 4 microphones, 3 cameras in each room.
+The movie crew also had 2 setup setups on sight with 8 monitors.
+They could make professional movie cuts of the talks directly after the talks.
+Just 30 minutes after a talk was finished it was uploaded directly into familiar movie services.
+You can find them on the [microXchg YouTube Channel](https://www.youtube.com/channel/UCGCbB8TPtYMQmJwYVogcPjg).
+And also on [InfoQ](https://www.infoq.com/microxchg), which is one of the best platform for tech talks on the internet as slides and speaker recordings are presented in two windows.
+As you can watch all recordings there I will only give a summary on one single talk
```

Comment:
As you can watch all recordings online I will only give a summary of one single talk that impressed me the most.